uktrade/data-hub-frontend
# RR-831-refactor-reminder-urls

## Description of change

- Move the logic for determining the default reminder link logic from notification bell component to the reminder component
- If no reminder type is selected, redirect to the default based on user feature flags

## Test instructions

This logic requires the use of `export-notifications` and the `investment-notifications` feature groups. 
Disabling `investment-notifications` and navigating to http://localhost:3000/reminders should redirect to the first export reminders link.
Enabling `investment-notifications` and navigating to http://localhost:3000/reminders should redirect to the first investment reminders link.

## Screenshots

### Before

![chrome_kuLQDnmHF8](https://user-images.githubusercontent.com/102232401/214088559-5c7801f8-21b1-4736-8ffb-19d2565706da.gif)


### After


![chrome_tIj7TlwJtt](https://user-images.githubusercontent.com/102232401/214088500-dfa24ff5-a009-4e23-bf35-91e6407a65f6.gif)

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
